packages/fetch-and-compile/test/fetch.test.ts

```diff
@@ -0,0 +1,81 @@
+import assert from "assert";
+import { describe, it } from "mocha";
+import Config from "@truffle/config";
+import { fetchAndCompile } from "../lib/index";
+import axios from "axios";
+import sinon from "sinon";
+const fixture: any = require("./fixture.js");
+
+function axiosMock(url: string, address: string, data: object) {
+  sinon.stub(axios, 'get').withArgs(url, {
+    params: {
+      module: "contract",
+      action: "getsourcecode",
+      address: address,
+      apikey: ''
+    },
+    responseType: "json",
+    maxRedirects: 50
+  }).returns(Promise.resolve({ data: data }))
+}
+afterEach(()=>
+    //@ts-ignore
+    //restoring stub
+    axios.get.restore()
```

Comment:
Can you indent only 2 spaces? In Truffle we never use anything other than 2 space indentation. Also in general I think we use "function" notation for functions rather than arrow functions in Mocha tests because that's what Mocha recommends. In other words,
`afterEach(function() {...` for the callbacks in Mocha's `it`s, `after`s, `beforeEach`s, etc. It is mostly to have a useful `this` value. Arrow functions don't bind `this` the same way that "function" syntax does.

Reply:
Oh, this is true, in Mocha you should use `function() { ... }` rather than arrow functions, I'd forgotten that.

Reply:
Sweet I'll change these and get ride of the arrow functions. Thanks.

Reply:
Note to be clear of course I don't mean that you shouldn't use arrow functions at *anywhere* in Mocha, just like, they shouldn't be used as the test functions used in an `it`, `describe`, `afterEach`, etc.

Reply:
Yup yup, I changed these as per the mocha docs. Thanks for the heads up.